src/key/subkey.js

```diff
@@ -87,37 +90,36 @@ class SubKey {
 
   /**
    * Returns the expiration time of the subkey or Infinity if key does not expire
-   * Returns null if the subkey is invalid.
-   * @param  {SecretKeyPacket|
-   *          PublicKeyPacket} primaryKey  The primary key packet
+   * Returns null if the subkey is invalid
```

Comment:
This is very nitpicky but personally I prefer the version with dots, to have proper grammar, so I would prefer adding a dot to the first line rather than removing the second one.